samples/wifi/softap/README.rst

```diff
@@ -0,0 +1,140 @@
+.. _wifi_softap_sample:
+
+Wi-Fi: Raw TX packet
+####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The SoftAP sample demonstrates how to start a nRF70 Series device in Software-enabled Access Point (SoftAP or SAP) mode.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+Overview
+********
+The sample enables a SoftAP mode and allows the devices to connect to it.
+The sample starts a Dynamic Host Configuration Protocol (DHCP) server for automatic IP address assignment to the connected devices.
+The sample lists the connected devices.
+
+You can enable the SoftAP mode by setting the below configuration options in the sample project configuration file:
+
+* :kconfig:option:`CONFIG_NRF700X_AP_MODE`: Enables Access point mode support.
+* :kconfig:option:`CONFIG_WPA_SUPP_AP`: Enables Access point support.
+
+.. note::
+
+   The SoftAP mode operation is dictated by regulatory requirements.
+
+You can set the regulatory domain using the below configuration option:
+
+* :kconfig:option:`CONFIG_SOFTAP_SAMPLE_REG_DOMAIN`: Sets the ISO/IEC alpha2 country code.
+
+For more information, see :ref:`nRF70_soft_ap_mode`.
+
+Configuration
+*************
+
+|config|
+
+Configuration options
+=====================
+
+The following sample-specific Kconfig options are used in this sample (located in :file:`samples/wifi/softap/Kconfig`):
+
+.. options-from-kconfig::
+
+IP addressing
+*************
+
+The sample starts DHCP server on SoftAP interface.
+The station devices should use DHCP to get an IP address from the virtual router.
+
+To specify the DHCP pool start address, you can edit the following configuration option in the :file:`prj.conf` file:
+
+.. code-block:: console
+
+  CONFIG_SOFTAP_SAMPLE_DHCPV4_POOL_START="192.168.1.2"
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/wifi/softap`
+
+.. include:: /includes/build_and_run_ns.txt
+
+To build for the nRF7002 DK, use the ``nrf7002dk_nrf5340_cpuapp`` build target.
+The following is an example of the CLI command:
+
+.. code-block:: console
+
+   west build -b nrf7002dk_nrf5340_cpuapp
+
+To build for the nRF7002 EK with nRF5340 DK, use the ``nrf5340dk_nrf5340_cpuapp`` build target with the ``SHIELD`` CMake option set to ``nrf7002ek``.
+The following is an example of the CLI command:
+
+.. code-block:: console
+
+   west build -b nrf5340dk_nrf5340_cpuapp -- -DSHIELD=nrf7002ek
```

Comment:
We should stop putting build commands, this is covered ni `Requirements` section. If necessary we can add a generic para on how to use that table.